samlconnection/client.go

```diff
@@ -0,0 +1,119 @@
+// Package samlconnection provides the SAML Connections API.
+package samlconnection
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/clerk/clerk-sdk-go/v2"
+)
+
+//go:generate go run ../cmd/gen/main.go
+
+const path = "/saml_connections"
+
+// Client is used to invoke the SAML Connections API.
+type Client struct {
+	Backend clerk.Backend
+}
+
+func NewClient(config *clerk.ClientConfig) *Client {
+	return &Client{
+		Backend: clerk.NewBackend(&config.BackendConfig),
+	}
+}
+
+type AttributeMappingParams struct {
```

Comment:
Note to reviewer: Not sure if we prefer to redefine this struct here in order to be explicit or re-use it from the clerk package